pyodide-build/pyodide_build/pypabuild.py

```diff
@@ -163,7 +163,24 @@ def make_command_wrapper_symlinks(symlink_dir: Path) -> dict[str, str]:
             symlink_path.unlink()
         try:
             pywasmcross_exe = shutil.which("_pywasmcross")
-            if pywasmcross_exe:
+            pywasmcross_exe_pyenv = None
+
+            # pyenv creates a shim that eventually calls the real executable,
```

Comment:
There are a whole bunch of tools that do roughly similar things to pyenv. Is it the only one that needs a workaround like this? Is there a good reason to privilege it over similar tools?

Reply:
>  Is there a good reason to privilege it over similar tools?

Not very much. It is just because I use pyenv locally and had trouble while working on this PR.

>  Is it the only one that needs a workaround like this?

No, I think tools like `pyenv`, `asdf`, will have similar problems, and I guess we need a wordaround for each tools :(

Reply:
What about vendoring the pywasmcross script into the xbuildenv?

Reply:
The point is that pyenv has some special logic that changes how `console_scripts` are handled at package install time?

Reply:
> The point is that pyenv has some special logic that changes how console_scripts are handled at package install time?

Yes, exactly.

> What about vendoring the pywasmcross script into the xbuildenv?

Sounds like a interesting idea. Can tar.bz2 archive preserve a file permission? We need execute permission.